### PR TITLE
DEV-2055: Sanitize LODManager.setWorldQualityDetail() parameter value

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Case: https://highfidelity.atlassian.net/browse/DEV-2055

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22005/HighFidelity-Beta-PR16372-3bcb305.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22005/HighFidelity-Beta-Interface-PR16372-3bcb305.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22005/HighFidelity-Beta-PR16372-3bcb305.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22005/HighFidelity-Quest-Beta-PR16372-3bcb305.apk)

File: interface/src/LODManager.cpp [hunk 1]
```diff
@@ -397,7 +397,7 @@ void LODManager::setWorldDetailQuality(float quality) {
     static const float MIN_FPS = 10;
     static const float LOW = 0.25f;
 
-    bool isLowestValue = quality == LOW;
```
Comment (samcake) [inline interface/src/LODManager.cpp]: I think the best approach for this one is to replace this varaible by an enum
which can take currently 3 value (low mid high)
and avoid all these complications of remapping a float

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22005/HighFidelity-Beta-PR16372-3bcb305.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22005/HighFidelity-Beta-Interface-PR16372-3bcb305.exe

File: interface/src/LODManager.cpp (hunk 1 above)
Comment (ctrlaltdavid) [reply interface/src/LODManager.cpp]: Done.

**Comment by ctrlaltdavid** [comment]
Notes:
- Settings > Graphics "Low", "Medium", or "High" setting won't be "remembered" from previous Interface version when this one is installed. (But they will be remembered between runs of this Interface version.) This is because the setting was implicitly stored in the desktop/HMD "target fps" float whereas now it's explicitly stored in its own enum value.
- In the Settings > Graphics dialog, "Custom" graphics setting and options aren't remembered between Interface runs. But this also happens in current RC. https://highfidelity.atlassian.net/browse/DEV-2478
- In the Settings > Graphics dialog, setting Low/Medium/High in desktop mode also affects the setting in HMD mode, and vice versa. Whereas, LODManager code indicates that they should be remembered separately. This also happens in current RC. https://highfidelity.atlassian.net/browse/DEV-2484

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22051/HighFidelity-Beta-PR16372-378c47d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22051/HighFidelity-Beta-Interface-PR16372-378c47d.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22052/HighFidelity-Beta-PR16372-6a2e67c.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22052/HighFidelity-Quest-Beta-PR16372-6a2e67c.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22052/HighFidelity-Beta-PR16372-6a2e67c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22052/HighFidelity-Beta-Interface-PR16372-6a2e67c.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22059/HighFidelity-Beta-PR16372-ea864e4.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22059/HighFidelity-Quest-Beta-PR16372-ea864e4.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22059/HighFidelity-Beta-PR16372-ea864e4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22059/HighFidelity-Beta-Interface-PR16372-ea864e4.exe

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22065/HighFidelity-Beta-PR16372-ea864e4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22065/HighFidelity-Beta-Interface-PR16372-ea864e4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22065/HighFidelity-Beta-PR16372-ea864e4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22065/HighFidelity-Beta-Interface-PR16372-ea864e4.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22065/HighFidelity-Beta-PR16372-ea864e4.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22065/HighFidelity-Quest-Beta-PR16372-ea864e4.apk)

**Pull request merged**